grpc/csharp-cubic/.gitignore

```diff
@@ -0,0 +1,349 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
```

Comment:
Wow, MS adds a lot of bloat! :smirk:

Reply:
I blindly took that from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore.  I hope that's ok.

Reply:
yes, my snark is directed at Microsoft, not you!